src/applications/mhv-landing-page/containers/Alerts.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+// import { CSP_IDS } from '~/platform/user/authentication/constants';
+
+import {
+  AlertMhvBasicAccount,
+  AlertMhvRegistration,
+  // AlertNotVerified,
+  AlertUnregistered,
+  AlertVerifyAndRegister,
+} from '../components/alerts';
+
+import {
+  hasMhvAccount,
+  hasMhvBasicAccount,
+  isAuthenticatedWithSSOe,
+  isLOA3,
+  isVAPatient,
+  showVerifyAndRegisterAlert,
+  signInServiceName,
+} from '../selectors';
+
+const Alerts = () => {
+  const userVerified = useSelector(isLOA3);
+  const vaPatient = useSelector(isVAPatient);
+  const userRegistered = userVerified && vaPatient;
+  const userHasMhvAccount = useSelector(hasMhvAccount);
+  const userHasMhvBasicAccount = useSelector(hasMhvBasicAccount);
+  const renderVerifyAndRegisterAlert = useSelector(showVerifyAndRegisterAlert);
+  const cspId = useSelector(signInServiceName);
+  const ssoe = useSelector(isAuthenticatedWithSSOe);
+
+  if (userHasMhvBasicAccount) {
+    return <AlertMhvBasicAccount ssoe={ssoe} />;
+  }
+
+  if (renderVerifyAndRegisterAlert) {
+    return <AlertVerifyAndRegister cspId={cspId} />;
+  }
+
+  // if (!userVerified && cspId === CSP_IDS.DS_LOGON) {
+  //   return <AlertNotVerified cspId={cspId} />;
+  // }
+
```

Comment:
Curious about why this alert is commented out?

Reply:
It should no longer apply. I left it commented so those who review this PR will know which VaAlert is going away, so it isn't a surprise. And, also since I'm not certain but there _could_ be DSLogon accounts < LOA1, so it would be easy to add back in.